lib/Conversion/TorchOnnxToTorch/DefaultDomainAtoF.cpp

```diff
@@ -147,6 +147,170 @@ void mlir::torch::onnx_c::populateDefaultDomainAtoF(
                   return success();
                 });
   // Add became forward compatible with Torch in version 7.
+  patterns.onOp(
+      "Adam", 1, [](OpBinder binder, ConversionPatternRewriter &rewriter) {
+        llvm::SmallVector<Value> operands;
+        if (binder.tensorOperandsList(operands))
+          return failure();
+
+        float alpha = 0.9;
+        float beta = 0.999;
+        float epsilon = 0.0;
+        float norm = 0.0;
+        float postNorm = 0.0;
```

Comment:
Is there a need to initialize all of them since the default value would be assigned during the binding itself? Also, can we declare them all in the same line?